pkg/apis/compliance/v1alpha1/compliancescan_types.go

```diff
@@ -29,6 +37,9 @@ const ComplianceScanLabel = "compliance.openshift.io/scan-name"
 // ScriptLabel defines that the object is a script for a scan object
 const ScriptLabel = "complianceoperator.openshift.io/scan-script"
 
+// KubeletConfigLabel defines that the object is a fetched KubeletConfig for a scan object
+const KubeletConfigLabel = "complianceoperator.openshift.io/scan-kubeletconfig"
```

Comment:
This label is just provided to be user-friendly, right? (Just asking, I think if it's the case then it's the right approach)

Reply:
Yes, also to be easier to find them when we need to delete them.